src/state-manager/TransactionQueue.ts

```diff
@@ -5608,7 +5608,12 @@ class TransactionQueue {
         this.clearDebugAwaitStrings()
 
         // eslint-disable-next-line security/detect-object-injection
-        const queueEntry: QueueEntry = this._transactionQueue[currentIndex]
+        const queueEntry: QueueEntry | undefined = this._transactionQueue[currentIndex]
+        if (queueEntry == null) {
```

Comment:
you could just do !queueEntry.  If you stick with null equality check we should use strict equals (===)

Reply:
Noted, and looked up why so I don't do this again. Pushed the change.